src/main/c/sslcontext.c

```diff
@@ -1396,70 +1396,13 @@ TCN_IMPLEMENT_CALL(void, SSLContext, setSessionTicketKeys0)(TCN_STDARGS, jlong c
 
 /* OpenSSL end */
 
-/*
- * Adapted from Android:
- * https://android.googlesource.com/platform/external/openssl/+/master/patches/0003-jsse.patch
- */
-const char* SSL_CIPHER_authentication_method(const SSL_CIPHER* cipher){
```

Comment:
This method was only used by our own `SSL_authentication_method` defined just below it.  I've renamed our `SSL_authentication_method` to `authentication_method` in order to avoid the naming conflict with OpenSSL's version. I then got rid of `SSL_CIPHER_authentication_method` which also conflicts with an OpenSSL method, and we don't need anymore anyway.